src/web/src/utils.js

```diff
@@ -380,3 +380,24 @@ export const exportScheduleToImage = (
       console.log("Oh No, Something Went Wrong!", error);
     });
 };
+
+export const getLongName = (department) => {
+  var dict = {
```

Comment:
STSO= Science & Technology Studies

Reply:
This subject code doesn't exist in `school-department-mapping.yaml`, should the change be reflected there too? I'm assuming it replaces STSH and STSS

Reply:
> This subject code doesn't exist in `school-department-mapping.yaml`, should the change be reflected there too? I'm assuming it replaces STSH and STSS

That is correct, and it should fall under HASS.